.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-Related Ticket: 
+Related Ticket: 
```

Comment:
```suggestion
Related Ticket: 
```
You've inserted a NBSP here somehow... so lets try this again.

Reply:
Thanks, I was trying to insert just a space but it was consistently trimmed in vs code, appreciate the suggestion.